spec/test.js

```diff
@@ -1,43 +1,55 @@
+/*global jasmine*/
 var selenium = require('selenium-webdriver'),
-    AxeBuilder = require('axe-webdriverjs');
+    AxeBuilder = require('axe-webdriverjs'),
+    driver, browser;
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 180000;
 
 describe('Selenium-aXe Tutorial', function() {
 
     // Open the Deque website in the browser before each test is run
     beforeEach(function(done) {
-        this.driver = new selenium.Builder()
-            .forBrowser('firefox')
-            .build();
+        driver = new selenium.Builder()
+                .forBrowser('chrome')
+                .usingServer('http://localhost:4444/wd/hub');
```

Comment:
Does this assume you have started Selenium standalone in another tab? If so we should update the README.